crates/frontend/src/pages/default_config/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod default_config;
```

Comment:
Can we stop doing these one line mod.rs files? We can just put all the code here

Reply:
Leave it as is for now